src/relay/op/tensor/transform.cc

```diff
@@ -358,23 +358,32 @@ bool StackRel(const Array<Type>& types,
   }
   const auto* param = attrs.as<StackAttrs>();
   const auto& first = Downcast<TensorType>(tensor_tuple->fields[0]);
-  // Sanity check: ndim and dtype.
   const int ndim = static_cast<int>(first->shape.size());
+
+  // Sanity check: axis
+  int axis = param->axis;
+  CHECK(-ndim <= axis && axis < ndim)
+    << "stack only accepts `axis` in [-ndim, ndim)"
+    << ", but got axis = " << axis
+    << ", and ndim = " << ndim;
+  axis = axis < 0 ? ndim + axis + 1 : axis;
```

Comment:
Shouldn't this be
```c++
axis = axis < 0 ? ndim + axis: axis;
```

Reply:
fixed.

Reply:
wait a min. This leads to a CI error on examples. This part of the code is actually adapted from https://github.com/dmlc/tvm/blob/master/src/relay/op/tensor/transform.cc#L377, though I didn't understand why +1 is needed here.

Reply:
I see. You can probably change it back.

Reply:
CI checks passed. Is it ready for merging?